src/test/java/org/apache/commons/text/ExtendedMessageFormatTest.java

```diff
@@ -416,6 +416,81 @@ public void testSetFormatsByArgumentIndex() {
         emf.setFormatsByArgumentIndex(new Format[]{new LowerCaseFormat(), new UpperCaseFormat()});
     }
 
+    @Test
+    public void testFailsToCreateExtendedMessageFormatTakingThreeArgumentsThrowsIllegalArgumentExceptionOne() {
```

Comment:
The name of this test method (and some of the ones below) seems misleading, because the name suggests that the three argument constructor is tested, but the tests calls the two argument constructor.